dev-requirements.txt

```diff
@@ -1,13 +1,74 @@
+#
+# This file is autogenerated by pip-compile with Python 3.10
+# by the following command:
+#
+#    pip-compile dev-requirements.in
+#
+cffi==1.15.1
+    # via cryptography
+click==8.1.6
+    # via
+    #   click-default-group
+    #   towncrier
+click-default-group==1.2.2
+    # via towncrier
 coverage==7.0.4
-tornado==6.2
-PySocks==1.7.1
+    # via -r dev-requirements.in
+cryptography==41.0.2 ; implementation_name != "pypy" or implementation_version >= "7.3.10"
```

Comment:
Looks like cryptography can't be split into separate versions with pip-compile :/